src/EditorFeatures/Test/Utilities/EditorServicesUtil.cs

```diff
@@ -16,10 +14,9 @@ public static class EditorServicesUtil
 
         private static IExportProviderFactory CreateExportProviderFactory()
         {
-            var assemblies = TestExportProvider
-                .GetCSharpAndVisualBasicAssemblies()
-                .Concat(new[] { typeof(EditorServicesUtil).Assembly });
-            return ExportProviderCache.GetOrCreateExportProviderFactory(ExportProviderCache.GetOrCreateAssemblyCatalog(assemblies, ExportProviderCache.CreateResolver()));
+            var catalog = TestExportProvider.GetCSharpAndVisualBasicAssemblyCatalog()
+                .WithParts(ExportProviderCache.GetOrCreateAssemblyCatalog(new[] { typeof(EditorServicesUtil).Assembly }, ExportProviderCache.CreateResolver()));
+            return ExportProviderCache.GetOrCreateExportProviderFactory(catalog);
```

Comment:
what's changing here?

Reply:
This is because of the change in TestexportProvider with changing type from IEnumerable<Assembly> to ComposableCatalog. 

@sharwell, as I remember, you are a stakeholder for this. If this is correct. Please decide if you want to include this in the PR or consider as a separate one. If I'm incorrect, I'd revert this and related changes.

---
In reply to: [232508962](https://github.com/dotnet/roslyn/pull/29016#discussion_r232508962) [](ancestors = 232508962)

Reply:
@JasonMalinowski, we're OK here. It is an improvement @sharwell suggested to include.

---
In reply to: [234823370](https://github.com/dotnet/roslyn/pull/29016#discussion_r234823370) [](ancestors = 234823370)